File: Add Code Here/PYTHON/linear_search.py
def linear_search(arr, target):
    for i in range(len(arr)):
        if arr[i] == target:
            return i  # Return the index where the target element is found
    return -1  # Return -1 if the target element is not in the list

# Example usage:
my_list = [10, 23, 45, 7, 15, 56, 89]
target_element = 15
result = linear_search(my_list, target_element)

if result != -1:
    print(f"Element {target_element} found at index {result}")
else:
    print(f"Element {target_element} not found in the list")

File: Add Code Here/Julia/Linear_Search.jl
function linear_search(arr, target)
    for (index, value) in enumerate(arr)
        if value == target
            return index  # Return the index if the target is found
        end
    end
    return -1  # Return -1 if the target is not found in the array
end

# Example usage
arr = [3, 1, 4, 1, 5, 9, 2, 6, 5, 3, 5]
target = 6
result = linear_search(arr, target)

if result == -1
    println("$target not found in the array.")
else
    println("$target found at index $result.")
end
